x-pack/test/fleet_api_integration/apis/agent_policy/agent_policy.ts

```diff
@@ -6,6 +6,7 @@
  */
 
 import expect from '@kbn/expect';
+import { expect as jestExpect } from 'expect';
```

Comment:
I'm curious why you imported this additional `expect`? was it an error or is it for some specific reason?

Reply:
There was no real reason. As I was developing I automatically went for the jest api. Not paying attention where "expect" came from, I though I was working with the jest api, and started getting type errors. So instead of switching the function that I used, I imported jest's expect. I removed jestExpect and I'm using kbn/expect now.